GNUmakefile

```diff
@@ -30,6 +30,7 @@ TAGGED_REPO := $(REPO):$(DOCKER_TAG)
 
 .PHONY: install
 install: operator-ui-autoinstall install-chainlink-autoinstall ## Install chainlink and all its dependencies.
+	git config core.hooksPath .githooks
```

Comment:
Any way we can opt out of this particular step? I know that was one of our concerns. Maybe we should have it as a separate make command

Reply:
What are the concerns, here?

We could make a symlink to `/.githooks/pre-push`, from `/.git/hooks/pre-push`.

We could check `/.git/hooks/`, and warn that we'll be ignoring it from now on, before doing the `git config`.

Reply:
Maybe I'm missing something here, but I know that some of the engineers didn't want hooks to be automatically installed.

Reply:
Ah, OK. Yeah, a separate make stanza is fine by me. Thanks.

Reply:
Maybe something like `if [ "$SKIP_HOOKS" eq "" ]; then`

Reply:
I second putting it in its own command over make install.  It follows the principle of least surprise, and this is definitely a surprise :P

Reply:
I've moved it to its own Makefile stanza, which has to be executed explicitly.